render/gles2/renderer.c

```diff
@@ -331,6 +331,86 @@ static bool gles2_read_pixels(struct wlr_renderer *wlr_renderer,
 	return glGetError() == GL_NO_ERROR;
 }
 
+static bool gles2_blit_dmabuf(struct wlr_renderer *wlr_renderer,
+		struct wlr_dmabuf_attributes *dst_attr,
+		struct wlr_dmabuf_attributes *src_attr) {
+	assert(!wlr_renderer->rendering);
```

Comment:
This assert should be moved in the common code, in `wlr_renderer_blit_dmabuf`, so that each renderer doesn't need to add it.